### PR TITLE
switch to axios

### LINKED ISSUE #17: Use a different network library so the sdk is usable in browser
The **request** and **request-promise** libraries do not work in the browser, (which is as intended) but it would be easy to replace them with a cross platform solution such as [axios](https://github.com/axios/axios). I've started working on this in a fork for my own use, but I'd love to upstream my change for everyone to enjoy.

### PR DESCRIPTION
**Pull request by somkun**
fixes #17

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,50 +1,49 @@
...
-		"@types/request-promise": "4.1.42",
-		"@types/gulp-mocha": "0.0.32",
-		"@types/chai": "4.1.7",
-		"@types/chai-as-promised": "7.1.0",
-		"@types/gulp": "4.0.5",
-		"@types/gulp-plumber": "0.0.32",
-		"@types/merge2": "1.1.4",
-		"@types/del": "3.0.1"
-	}
+  "name": "scryfall-sdk",
```
Comment (Yuudaari) [inline package.json]: Please fix the indentation.

File: src/Scry.ts [hunk 2]
```diff
@@ -54,19 +54,21 @@ async function queryApi<T> (
...
-	}).catch(({ response }) => {
-		err = response.body;
+	const path = Array.isArray(apiPath)
+		? (apiPath = apiPath.join("/"))
+		: apiPath.toString();
+	const url = `${endpoint}/${path}`;
+
+	const result = await (post
+		? Axios.post(url, post, { params: query })
+		: Axios.get(url, { params: query })
```
Comment (Yuudaari) [inline src/Scry.ts]: Use the plain function call so that it's 1:1 to the request code https://www.npmjs.com/package/axios#axiosconfig

**Comment by Yuudaari** [comment]
Looks good, thanks for the contribution! I'll get a new build out today or tomorrow. How would you like to be credited in the changelog? Is "somkun" linking to your github profile good?

**Comment by annich-MS** [comment]
works for me, thanks for the quick turn around :)

**Pull request merged**